### PR TITLE
612 revise operations form

### PR DESCRIPTION
**Pull request by marcellmueller**
Implements #612

### CODE REVIEW

File: client/app/utils/jsonSchema/operations.ts
Comment (andrea-williams) [inline client/app/utils/jsonSchema/operations.ts:61]: I'd recommend deleting the `process_flow_diagram` and `boundary_map` parts entirely - I don't think we'll ever be using them

File: client/app/utils/jsonSchema/operations.ts
Comment (andrea-williams) [inline client/app/utils/jsonSchema/operations.ts:430]:
```suggestion
```

File: client/app/utils/jsonSchema/operations.ts
Comment (andrea-williams) [inline client/app/utils/jsonSchema/operations.ts:431]:
```suggestion
```

File: client/app/utils/jsonSchema/operations.ts
Comment (andrea-williams) [inline client/app/utils/jsonSchema/operations.ts:441]: I don't think we need lines 439-441 anymore, do we?

File: client/app/utils/jsonSchema/operations.ts
Comment (andrea-williams) [inline client/app/utils/jsonSchema/operations.ts:508]: can just entirely delete lines 503-508

File: client/app/utils/jsonSchema/operations.ts
Comment (andrea-williams) [inline client/app/utils/jsonSchema/operations.ts:463]: comment out lines 461-463?

**Review by andrea-williams — APPROVED** [review]
Some small suggestions for extra lines that can be commented out or deleted entirely. 

On behalf of @Sepehr-Sobhani and his current rebasing miseries, I'd ask that you don't merge this PR into develop until he's finished fixing his PR and got it merged, since the changes made in this PR to models.py will impact Sep's PR.

**Comment by marcellmueller** [comment]
> Some small suggestions for extra lines that can be commented out or deleted entirely.
> 
> On behalf of @Sepehr-Sobhani and his current rebasing miseries, I'd ask that you don't merge this PR into develop until he's finished fixing his PR and got it merged, since the changes made in this PR to models.py will impact Sep's PR.

Awesome thanks. I've cleanup up all the unneeded commented code in the operations schema you pointed out and will wait until Sepehr's PR gets merged.

**Pull request merged**
